Abs.lua

```diff
@@ -1,3 +1,5 @@
+local THNN = require('nn.THNN')
```

Comment:
@fmassa If the tensor metatable-'trick' is used there is no need for `require THNN` in each file (only once in init.lua).

In general I wanted to avoid further global namespace pollution - I think torch has there been a litte bit sloppy in the past.